be/src/storage/data_dir.cpp

```diff
@@ -575,9 +575,9 @@ Status DataDir::update_capacity() {
 
 bool DataDir::capacity_limit_reached(int64_t incoming_data_size) {
     double used_pct = (_disk_capacity_bytes - _available_bytes + incoming_data_size) / (double)_disk_capacity_bytes;
-    int64_t left_bytes = _disk_capacity_bytes - _available_bytes - incoming_data_size;
+    int64_t left_bytes = _available_bytes - incoming_data_size;
```

Comment:
https://github.com/StarRocks/starrocks/blob/1ceb126e4cb13645e3bcbd2edc366dd0d96f7b03/be/src/common/config.h#L562C12-L562C12  

modify the comment?